### PR TITLE
chore(website): Use client side submit

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Fixes https://github.com/cloudquery/cloudquery/issues/2682. Will test on the deploy preview, then move to ready to review

### CODE REVIEW

File: website/components/Footer.tsx [hunk 1]
```diff
@@ -142,50 +142,36 @@ export function Footer() {
...
-      onSubmit={subscribe}
+      action="https://cloudquery.us1.list-manage.com/subscribe/post"
+      method="POST"
+      onSubmit={handleSubmit}
     >
       <label htmlFor="email-address" className="sr-only">
         Email address
       </label>
       <div>
+        <input type="hidden" name="u" value="5a433922a8db867a38c8a4a8e" />
```
Comment (erezrokah) [inline website/components/Footer.tsx:164]: Taken by following https://mailchimp.com/help/host-your-own-signup-forms/ (explained in the issue0

File: website/components/Footer.tsx [hunk 2]
```diff
@@ -142,50 +142,36 @@ export function Footer() {
...
-      setMessage(error);
-
-      return;
-    }
-
     router.push("/confirm");
   };
 
   return (
     <form
```
Comment (erezrokah) [inline website/components/Footer.tsx:153]: This is now an uncontrolled component, so no need to track the email input and errors

File: website/components/Footer.tsx [hunk 3]
```diff
@@ -145,21 +145,19 @@ function SubmitForm() {
...
+    const params = new URLSearchParams();
+    params.append("EMAIL", email);
+    params.append("u", "5a433922a8db867a38c8a4a8e");
+    params.append("id", "4a9aca3509");
+    params.append("c", "?");
 
-      return;
-    }
+    await fetch(
+      `https://cloudquery.us1.list-manage.com/subscribe/post-json?${params.toString()}`,
```
Comment (erezrokah) [inline website/components/Footer.tsx:158]: We need `post-json` and `no-cors` due to a Mailchimp limitation, which means we can't get the text response and check for errors.
If we want to handle errors we need to use https://github.com/camsong/fetch-jsonp to handle this `jsonp` request.
I don't expect this to fail so I think that's OK (it only failed for me when I tried to submit too many times due to rate limiting on Mailchimp side)

**Review by yevgenypats — APPROVED** [review]
very nice.

**Pull request merged**